python/tests/test_pytest.py

```diff
@@ -5,7 +5,8 @@
 
 def expected_messages(output):
     assert '= 4 failed, 2 passed, 1 skipped, 1 xpassed, 6 error in' in output, output
-    assert 'integrations/pytest/test_all.py:27: skipping test message' in output, output
+    assert ('integrations/pytest/test_all.py:27: skipping test message' in output
+            or 'integrations/pytest/test_all.py:28: skipping test message' in output), output
```

Comment:
This change is because of a bug fix in pytest 3.2. Before then, the skip line was recorded as 1 line above the `pytest.skip` call. So the test's skip message will vary depending on the version of pytest it's running on.